src/RPC/utils.ts

```diff
@@ -371,11 +382,14 @@ function clientInputTransformStream<I extends JSONValue>(method: string) {
   });
 }
 
-function clientOutputTransformStream<O extends JSONValue>() {
+function clientOutputTransformStream<O extends JSONValue>(
```

Comment:
I don't get the distinction between `rpc/utils.ts` and `rpc/middleware.ts`. I think they should be unified.

Reply:
They are kind of the same thing. I created `middleware.ts` to hold the middleware transforms  since they are not really used internally by the RPC but used to configure the middleware when constructing the `RPC`. I can re-combine them, but to me it's neater to have a `middleware.ts` for all your middleware needs.

Reply:
Like I said it should be either a directory with an `index.ts` or a utils.

Reply:
Suggest using `middleware/*` where `*` is name of middleware - use camel cased names.

Reply:
Fixed.